caproto/asyncio/repeater.py

```diff
@@ -84,26 +104,39 @@ def connection_lost(self, exc):
 async def start_datagram_proxy(bind, port):
     loop = asyncio.get_event_loop()
     protocol = ProxyDatagramProtocol()
-    return await loop.create_datagram_endpoint(
-        lambda: protocol, local_addr=(bind, port))
+    try:
+        return await loop.create_datagram_endpoint(
+            lambda: protocol, local_addr=(bind, port))
+    except OSError as ex:
+        if 'Address already in use' in str(ex):
+            raise RepeaterAlreadyRunning(str(ex))
+        else:
+            raise
 
 
 def main():
-    import logging
-    logging.getLogger('caproto').setLevel(logging.INFO)
     loop = asyncio.get_event_loop()
     addr = ('0.0.0.0', os.environ.get('EPICS_CA_REPEATER_PORT', 5065))
-    print("Starting datagram proxy on {}...".format(addr))
+    print("[repeater] Starting datagram proxy on {}...".format(addr))
     coro = start_datagram_proxy(*addr)
-    transport, _ = loop.run_until_complete(coro)
-    print("Datagram proxy is running...")
+
+    try:
+        transport, _ = loop.run_until_complete(coro)
+    except RepeaterAlreadyRunning as ex:
+        print('[repeater] Another repeater is already running; exiting')
+        return
+
+    print("[repeater] Datagram proxy is running...")
     try:
         loop.run_forever()
     except KeyboardInterrupt:
         pass
-    print("Closing transport...")
+    print("[repeater] Closing transport...")
     transport.close()
     loop.close()
 
+
 if __name__ == '__main__':
+    import logging
```

Comment:
I believe so - but it got forgotten in the mix of things. Repeater does still need some work outside of this.